todo/src/index.js

```diff
@@ -0,0 +1,22 @@
+import React from 'react';
+import ReactDOM from 'react-dom';
+import { createStore } from 'redux';
+import { Provider } from 'react-redux';
+import reducer from './reducers';
+
+import './index.css';
+import App from './App';
+// import * as serviceWorker from './serviceWorker';
+
+const store = createStore(reducer);
+
+ReactDOM.render(
+    <Provider store={store}>
+        <App />
+    </Provider>,
+    document.getElementById('root'));
```

Comment:
It's preferred to keep indentation even, so for the opening parenthesis on line 13, we want one below on line 18. So:
```javascript
ReactDOM.render(
    <Provider store={store}>
        <App />
    </Provider>,
    document.getElementById('root')
);
```